package.json

```diff
@@ -28,23 +28,26 @@
   "dependencies": {
     "csrf": "^3.0.2",
     "debug": "^2.1.2",
+    "deep-clone-merge": "^1.2.0",
     "express": "^4.12.2",
-    "hmpo-form-controller": "^1.1.0",
     "hmpo-model": "^1.0.0",
     "hogan.js": "^3.0.2",
     "i18n-lookup": "^0.1.0",
     "moment": "^2.17.1",
     "underscore": "^1.8.2"
   },
   "devDependencies": {
-    "chai": "^3.5.0",
+    "chai": "^4.1.2",
```

Comment:
Travis is failing in `node v6` and `node v7` due to:
`Error: Invalid Chai property: function`

Reply:
These have been fixed (`to.be.a.function;` changed to `to.be.a('function')`, etc)

Reply:
:+1: